recipes/modelbase/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "modelbase" %}
+{% set version = "1.3.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 662682567ecae226718a634ab5162f49d62d262b9e730f58cec09bcdea7f7417
+
+build:
+  number: 0
+  skip: True  # [py2k or py<37]
```

This looks like pure Python, so it should be able to be noarch Python:
```suggestion
  number: 0
  noarch: python
```